lib/android-helpers.js

```diff
@@ -266,5 +266,25 @@ helpers.unlock = async function (adb) {
   });
 };
 
+helpers.initDeviceAndGetDefaultIME = async function (adb, opts) {
```

Comment:
I think having the word "and" in a function name is an indicator that the function is doing too much.
What does getting the default IME have to do with initializing the device?

Can we break this into two functions? one that gets the default IME and another that initializes the device?